app/views/pages/faqs.html.erb

```diff
@@ -6,6 +6,270 @@
   <div class="usa-width-three-fourths usa-layout-docs-main_content">
     <h1><%= @page_title %></h1>
 
-    <%# TODO Add page content %>
+    <%# TODO Hide the checkboxes when the screen is too narrow %>
+    <div class="usa-grid" id="faq-filters">
+
+      <div class="usa-width-one-third">
+        <fieldset class="usa-fieldset-inputs usa-sans">
+          <ul class="usa-unstyled-list">
+            <li>
+              <input id="claims-checkbox" type="checkbox" value="claims">
+              <label for="claims">Claims</label>
```

Comment:
@jamesatheyDDS Related to this, I noticed the UI doesn't match [the USWDS' checkbox form controls demo page](https://standards.usa.gov/components/form-controls/#checkboxes). Long, dumb story short: I made a mistake in v1.3.0 of the uswds-rails gem and it needs updating:

```bundle update uswds-rails --full-index```

With the update, you'll get the proper UI and (with the updated `for` attribute values) you should still get the desired behavior.